### PR TITLE
Add org.emilien.Password

### PR DESCRIPTION
**Pull request by elescoute**

### CODE REVIEW

File: org.emilien.Password.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "password",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/elescoute/password-for-gnome.git"
```
Comment (LeandroStanger) [inline org.emilien.Password.json:33]:
```suggestion
                    "url" : "https://github.com/elescoute/password-for-gnome.git",
                    "commit": "68e5b9673a9f8c69903d2917fdea21c6e1b313b9"
```

**Comment by LeandroStanger** [comment]
bot, build org.emilien.Password

File: org.emilien.Password.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "modules" : [
+        {
+            "name" : "password",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/elescoute/password-for-gnome.git",
+                    "commit": "68e5b9673a9f8c69903d2917fdea21c6e1b313b9"
```
Comment (LeandroStanger) [inline org.emilien.Password.json:34]:
```suggestion
                    "commit": "236e587e2c2c6fde72bc7a99317d3d74937cdb2a"
```

**Comment by LeandroStanger** [comment]
bot, build org.emilien.Password

**Comment by LeandroStanger** [comment]
bot, build org.emilien.Password

**Comment by LeandroStanger** [comment]
bot, build org.emilien.Password

**Comment by LeandroStanger** [comment]
bot, build org.emilien.Password

**Comment by nedrichards** [comment]
bot, build org.emilien.Password

**Comment by elescoute** [comment]
It's ready for review.

**Pull request closed without merging**